webpack.config.js

```diff
@@ -284,6 +284,7 @@ const webpackConfig = {
 	plugins: _.compact( [
 		new webpack.DefinePlugin( {
 			'process.env.NODE_ENV': JSON.stringify( bundleEnv ),
+			'process.env.GUTENBERG_PHASE': JSON.stringify( 1 ),
```

Comment:
What do we need this for? :thinking:

Reply:
It's the same as the following line, Gutenberg depends on `process.env` at runtime 😞 

https://github.com/WordPress/gutenberg/pull/16679